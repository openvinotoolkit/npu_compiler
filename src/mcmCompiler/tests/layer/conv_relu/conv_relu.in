version https://git-lfs.github.com/spec/v1
oid sha256:87019afd946cf573785478dc1440dfaead37ba68079b0d9dff4191775050df2a
size 32
