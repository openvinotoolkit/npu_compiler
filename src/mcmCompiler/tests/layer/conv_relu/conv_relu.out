version https://git-lfs.github.com/spec/v1
oid sha256:7ea7187532e1d4da5a3063ae81b4f26f8856cec2209c5b638b6be2b4669475e6
size 2
